orttraining/orttraining/training_api/include/optimizer.h

```diff
@@ -64,10 +64,12 @@ struct Optimizer {
 
   Status GetStateDict(OptimizerCheckpointState& optimizer_checkpoint_states);
 
+  Status LoadStateDict(OptimizerCheckpointState& optimizer_checkpoint_states);
+
   int64_t GetStep() const {
     return optimizer_state_.step;
   }
-  
+
   Status SetLearningRate(float lr) {
```

Comment:
are we allowing users to call optimizer->SetLearningRate()? If not, need to make this method private

Reply:
`GetStep` can also be private.

Reply:
Fixed both.